packages/react/src/tokens/palettes/palette.ts

```diff
@@ -0,0 +1,151 @@
+export default {
+  brand: {
```

Comment:
Does it make sense to name it 'colors' and have all the colors and pallets inside?

Reply:
Yes, however, in Figma, the colors are separated in this way, and in my opinion, it looks more organized and easier to maintain. What are the benefits of having all colors in only one object?

Reply:
From a developer perspective, it is more easy to pass a `theme.colors` objects instead of `theme.brand` + `theme.neutral`etc...

We don't need to follow the organisation of Figma and could simplify a little bit.